lib/templates/single_file.eex

```diff
@@ -4,6 +4,26 @@
     <div class="viz-comment"><%= comment %></div>
 
     <h2>Routes</h2>
+    <% nil_stops_0 = {nil, 0} %>
+    <% nil_stops_1 = {nil, 1} %>
+    <% ns0 = vehicles_by_stop[nil_stops_0] || [] %>
+    <% ns1 = vehicles_by_stop[nil_stops_1] || [] %>
+    <% Map.delete(vehicles_by_stop, nil) %>
```

Comment:
This line doesn't do anything, right? `Map.delete/2` doesn't mutate the underlying map but rather returns a new one without that key.